lib/invoice.rb

```diff
@@ -26,4 +24,16 @@ def update_status(status)
   def update_time
     @updated_at = Time.now
   end
+
+  def created_at
+    return @created_at if @created_at.instance_of?(Time)
```

Comment:
Are these escape clauses for rspec?

Reply:
@NoahZinter for the spec harness -- they give us inputs in both Time and String formats, so it has to deal with either one coming in but always return a Time object